extensions/wikia/CreatePage/CreatePage.i18n.php

```diff
@@ -16,7 +16,7 @@
 	'createpage_cant_edit' => 'Could not perform edit',
 	'createpage-dialog-title' => 'Add New Page',
 	'createpage-dialog-message2' => 'Write your page title here:',
-	'createpage-dialog-message3' => 'We currently have {{NUMBEROFARTICLES}} pages on {{SITENAME}}. We need to check if the name is available.',
+	'createpage-dialog-message3' => 'We currently have {{NUMBEROFARTICLES}} {{PLURAL:$1|page|pages}} on {{SITENAME}}. We need to check if the name is available.',
```

Comment:
You're right - uploading them to crowdin removed previous translations. I filled them with their previous versions (which disregarded plural forms) and issued a ticket for LOC team asking them to update this string's translations.